docs/deploying-airbyte/on-cloud.md

```diff
@@ -8,7 +8,7 @@ Airbyte Cloud requires no setup and can be immediately run from your web browser
 
 **1. Using the Airbyte Cloud invite link provided to you, click `Sign up`, and verify your email.**
 
-If you don't have an invite, sign up [here!](https://airbyte.com/cloud-waitlist)
+<!-- If you don't have an invite, sign up [here!](https://airbyte.com/cloud-waitlist) -->
```

Comment:
@Amruta-Ranade I just commented this link out for historical purposes since this [page](https://docs.airbyte.com/deploying-airbyte/on-cloud/) is not displayed in the sidebar for users anyways.

Reply:
You can delete it